mantidproject/mantid
# Bug fix for muon results table and coadd runs

**Description of work.**
The results table of muon analysis was returning zeros in some of the cells as it was casting strings into numbers. 
**Report to:** Matthew and Keith. 

**To test:**
Open muon analysis
load a run
go to the settings tab
make sure multiple fitting is on
go to the data analysis tab
add a fitting function
in the runs box add a few more (e.g. 60212-6)
Make sure co-add is selected
Change the selected groups/pairs to groups
Do a fit
Go to the results table tab
Select multiple (middle of page)
Pick some sample logs (e.g. groups, sample_temp)
Click create table
Check it has sensible values

Click simultaneous (middle of page)
Click create table
check it has sensible values


Fixes #24237. 



In release notes
---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
